src/sequence_state.cc

```diff
@@ -146,15 +199,24 @@ SequenceStates::Initialize(
         state_size =
             triton::common::GetByteSize(state.second.data_type(), dims);
       }
-      data = std::make_shared<AllocatedMemory>(
-          state_size, TRITONSERVER_MEMORY_CPU, 0);
+      if (use_growable_memory) {
+        std::unique_ptr<GrowableMemory> growable_memory;
+        RETURN_IF_ERROR(GrowableMemory::Create(
```

Comment:
Could we remove create and use a constructor for growable memory?

Reply:
The reason we're using Create is to communicate any error that might have happened during the creation process. It is not possible to return an error using constructor.